canonical-web-and-design/maas-ui
# Take action statuses

## Done
- Display processing state for remaining actions in the take action menu.

## QA
- Open the machine list.
- Select some machines and use the take action menu to do different actions.
- Each action should show the processing status of the machines.

## Fixes
Fixes: https://github.com/canonical-web-and-design/maas-ui/issues/1010.
